templates/docs/examples/patterns/grid/_cve-table.html

```diff
@@ -0,0 +1,323 @@
+<table>
```

Comment:
When moving examples from `patterns` to `layouts` this should be moved as well.

Reply:
done